### PR TITLE
saucectl: init at 0.183.0

### LINKED ISSUE #336700: Package request: saucectl
**Project description**

The **saucectl** project is a command line interface for the test platform Saucelabs that orchestrates test execution in various frameworks.

**Metadata**

- homepage URL: [https://docs.saucelabs.com/dev/cli/saucectl/](https://docs.saucelabs.com/dev/cli/saucectl/)
- source URL: [https://github.com/saucelabs/saucectl](https://github.com/saucelabs/saucectl)
- license: Apache License, Version 2.0
- platforms: Windows Linux, macOS

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by luftmensch-luftmensch**
## Description of changes
Closes  #336700

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by luftmensch-luftmensch** [comment]
@brianmcgee Should be good to go now. Somehow this particular test does not fails on my machine previously

**Comment by brianmcgee** [comment]
Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>saucectl</li>
  </ul>
</details>

File: pkgs/by-name/sa/saucectl/package.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "-X github.com/saucelabs/saucectl/internal/version.Version=${version}"
+    "-X github.com/saucelabs/saucectl/internal/version.GitCommit=${version}"
+  ];
+
+  vendorHash = "sha256-SQveLJzicQSCBwfV3eZyXkArO1wLly1cCvol9PbeEV0=";
+
+  checkFlags = [ "-skip=^TestNewRequestWithContext$" ];
+
+  meta = {
+    description = "Command line interface for the Sauce Labs platform";
```
Comment (brianmcgee) [inline pkgs/by-name/sa/saucectl/package.nix:30]: Super minor nit.

```suggestion
    description = "Command line interface for the Sauce Labs platform.";
```

**Review by brianmcgee — APPROVED** [review]

File: pkgs/by-name/sa/saucectl/package.nix (hunk 1 above)
Comment (luftmensch-luftmensch) [reply pkgs/by-name/sa/saucectl/package.nix:30]: @brianmcgee  If I recall correctly there was a new standard that drops the dot at the end of the sentence

**Comment by picnoir** [comment]
@GrahamcOfBorg eval

**Comment by brianmcgee** [comment]
Is this PR hung on the 2 Darwin checks showing as queued? If so, is this stalled or a consequence of constrained resources?

I'd like to understand better how the review process works. Are there docs outlining it? I've never learnt how the 'sausage was made' and would like to get more involved in reviewing PRs.

**Comment by brianmcgee** [comment]
> I'd like to understand better how the review process works. Are there docs outlining it?

I should have just read https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md before writing that :man_facepalming:

**Pull request merged**